tests/Equinox.EventStore.Integration/StoreIntegration.fs

```diff
@@ -365,7 +367,53 @@ type Tests(testOutputHelper) =
         let! _ = service.Read cartId
         test <@ singleBatchBackwards @ batchBackwardsAndAppend @ singleBatchBackwards = capture.ExternalCalls @>
     }
-#endif
+    #else
+    [<AutoData(SkipIfRequestedViaEnvironmentVariable="EQUINOX_INTEGRATION_SKIP_EVENTSTORE")>]
+    let ``Can roundtrip against Store, correctly snapshotting to avoid redundant reads`` (ctx, skuId) = Async.RunSynchronously <| async {
+        #if NET
+        use _ = source.StartActivity("Can roundtrip against Store, correctly snapshotting to avoid redundant reads")
+        #endif
+        let log, capture = output.CreateLoggerWithCapture()
+        let! client = connectToLocalStore log
+        let batchSize = 10
+        let context = createContext client batchSize
+        let service = Cart.createServiceWithCompaction log context
```

Comment:
differnt helper with different name so reader can identify its a different strategy at a glance
(maybe compaction term in this codebase should become embedded, and the other one termed siblingStream or something like that; conveying 'not embedded')